doc/api/inspector.md

```diff
@@ -52,6 +52,21 @@ parameter usage.
 
 Return the URL of the active inspector, or `undefined` if there is none.
 
+```js
+// Node process was called with --inspect flag
+require('inspector').url()
+'ws://127.0.0.1:9229/whatever'
+
+// Node process was called with --inspect=localhost:3000 flag
+require('inspector').url()
+'ws://localhost:3000/whatever'
```

Comment:
This isn't supposed to be a line of executable code, right? It's the value of the previous line?

```suggestion
// 'ws://localhost:3000/whatever'
```